### ISSUE TITLE
ILAsm should support full signing

### ISSUE THREAD

**Issue by ericstj** [dotnet/coreclr#22862]
Roslyn can do full-signing on CoreCLR, ILAsm should do the same.  https://github.com/dotnet/roslyn/issues/8210

/cc @jkotas

**Comment by jkotas** [comment]
I do not think it makes sense to build full strong name signing into .NET Core ilasm.

If we need a full-signing in .NET Core build tooling for some reason, it should be built as separate tool.

**Comment by ericstj** [comment]
I noticed this when playing around with ILproj projects.  Our default is sign new assemblies with the open key which contains a public/private pair and this failed.  We can workaround it for sure, but I felt like the "right fix" was to address it in the tool.
